src/app/participant-page/participant-page.component.ts

```diff
@@ -109,6 +110,9 @@ export class ParticipantPageComponent implements OnInit, OnDestroy {
 
   private payload = {};
 
+  readonly kitFieldsDict = {'DATE_KIT_RECEIVED': 'receiveDate', 'DATE_KIT_SENT': 'scanDate', 'KIT_TYPE_TO_REQUEST': 'kitType'};
```

Comment:
could we get that info also from the instance_settings table, so that if another study would have type sample in their fields we could get it from there? or maybe we should add that to the form directly?